src/components/pages/SingleReview.css

```diff
@@ -35,4 +35,12 @@
   object-fit: cover;
   border-radius: 10px;
   margin-top: 1rem;
+  color: #333;
+}
+.comments-section {
+  background-color: #f2f2f2;
+  border-radius: 10px;
+  padding: 2rem;
+  text-align: left;
+  max-width: 800px;
```

Comment:
Currently the font colour is the same as the background so the text is not visible at the comments. Also the text for the individual reviews is barely readable for similar reason. Please see below:
<img width="966" alt="Screenshot 2023-06-02 at 08 38 08" src="https://github.com/alexmc2/nc-games/assets/54496822/9256b390-3887-4de7-ac6e-53064cd7813a">